src/components/book.js

```diff
@@ -1,9 +1,19 @@
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/book/bookSlice';
+import { getBookItems, removeBookItem } from '../redux/book/bookSlice';
 
 /* eslint-disable react/prop-types */
-const Book = ({ id, title, author }) => {
+/* eslint-disable camelcase */
+const Book = ({ item_id, title, author }) => {
```

Comment:
- [ ] I guess the item_id made you disable linters. To fix this issue, you can reassign the data coming from the api to your own state so that you will be able to use the variable name of your choice.

e.g 

in your reducer, you can do (this is just an example) :
```
{
"itemId": item_id,
title,
author
}
```

Reply:
Thanks let me try to make the changes